swift/vapor-framework/Package.swift

```diff
@@ -10,7 +10,7 @@ let package = Package(
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(url: "https://github.com/vapor/vapor.git", .upToNextMinor(from: "4.15.1"))
+        .package(url: "https://github.com/vapor/vapor.git", .upToNextMinor(from: "4.17.0"))
```

Comment:
We might as well just jump straight to 4.18 🙃

Reply:
Argh, you are too quick to release 🚀

Reply:
@0xTim is the bug about not having the directory named as the framework name solved in `SwiftPM` ?